spec/unit/sidekiq/retryable_jobs_filter_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.2.2')
+  require 'sidekiq'
+  require 'sidekiq/cli'
+  require 'airbrake/sidekiq'
+
+  RSpec.describe "airbrake/sidekiq/retryable_jobs_filter" do
+    subject(:filter) { Airbrake::Sidekiq::RetryableJobsFilter.new }
```

Comment:
Why `subject` instead of `let`? We don’t seem to take the advantage of it anywhere, so `let` seems to be the right fit here.

Reply:
Semantics, I guess? The `filter` is still the system under test so `subject` seems better than `let` for me. 

I noticed now that most of the specs are using `subject { described_class.new }` and I can adopt that form if you think it's better

Reply:
Unfortunately, RSpec doesn't provide any concrete pattern for this case. Same with BetterSpecs: http://www.betterspecs.org/#subject

They just say "you can use this" and that's it. Anyways, it seems to be a minor thing, so let's leave as is :)